### PR TITLE
fix(meet): restore prerender suspense boundary

### PR DESCRIPTION
**Pull request by vhpx**
## Summary

- wrap Meet's request-aware locale provider tree in Suspense
- add a structural regression test that keeps Nuqs and app providers inside the boundary

## Why

Production workflow 31456502280 built and deployed every affected application except Meet. Next.js Cache Components rejected `/[locale]` during prerender because runtime/request data escaped a Suspense boundary. Comparable satellite root layouts already use this boundary.

## Validation

- focused Meet layout test: 3 passed
- Meet type-check: passed
- Biome on both changed files: passed
- git diff check: passed
- repository check: Meet-related type-check and Biome passed; tests remain blocked by the unrelated finance-core module-resolution baseline

Local application builds were not run because repository policy requires an explicit build request. The PR and production deployment workflows are the authoritative build verification gates.

---
## Summary by cubic
Restored and narrowed a Suspense boundary around Meet’s request-aware server layout so `/[locale]` prerenders correctly without leaking request data.

- **Bug Fixes**
  - Wrapped `ServerLayout` in `<Suspense>` in `apps/meet/src/app/[locale]/(app)/layout.tsx`.
  - Added tests to assert the boundary surrounds `ServerLayout`, keeps the theme provider outside root boundaries, and preserves `NuqsAdapter` before app `Providers`.
  - Aligns Meet’s layout with other apps that already use this boundary.

<sup>Written for commit e82cda698a4e8f028eaee0cd57ca46186765b36d. Summary will update on new commits.</sup>

<a href="https://cubic.dev/pr/tutur3u/platform/pull/5117?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

### CODE REVIEW

File: apps/meet/src/app/[locale]/layout.tsx [hunk 1]
```diff
@@ -66,9 +66,11 @@ export default async function RootLayout({ children }: Props) {
       >
         <VercelAnalytics />
         <VercelInsights />
-        <NuqsAdapter>
-          <Providers appName={siteConfig.name}>{children}</Providers>
-        </NuqsAdapter>
+        <Suspense>
```
[the remark this answers is not in the document]
Comment (vhpx) [reply apps/meet/src/app/[locale]/layout.tsx]: Addressed in e82cda698a. The root-level boundary was removed so the shared ThemeProvider remains outside Suspense. The boundary now wraps the narrower request-aware ServerLayout that calls getTranslations, and the structural test verifies both the scoped boundary and the root no-Suspense contract. Focused tests pass 4/4 and the Meet type-check passes.
